src/service-worker/service-worker.ts

```diff
@@ -13,7 +13,7 @@ import { clientsClaim } from "workbox-core";
 import { ExpirationPlugin } from "workbox-expiration";
 import { precacheAndRoute, createHandlerBoundToURL } from "workbox-precaching";
 import { registerRoute } from "workbox-routing";
-import { StaleWhileRevalidate } from "workbox-strategies";
+import { CacheFirst } from "workbox-strategies";
```

Comment:
Why are we changing strategies?

Reply:
Assets are served from the cache storage using CacheFirst strategy....if they are not found in the cache, they are requested from the server then cached and the response from the server is served initially.

Reply:
Thanks for the explanation - but how is this different from stale-while-revalidate?

Reply:
While the implementation details under the hood are quite similar for both, what I observed is that the fonts and SVG are loading much faster in this version as compared to the zinzen.vercel.app. @tijlleenders Can you also do the same comparison on, say regular 2G, and check if that's true

Reply:
@GithinjiHans Can you tell me what the theoretical difference is between the two caching strategies CacheFirst and StaleWhileRevalidate?

Reply:
Stalewhile-revalidate also serves assets from cache but the assets in the cache are checked if they're upto date, if not they are not upto date they're updated and the latest version is served mostly used in live apps. Thats the main difference between Stalewhile-revalidate and Cache first.

Reply:
Ok - so we want [stale-while-revalidate](https://developer.chrome.com/docs/workbox/modules/workbox-strategies/#stale-while-revalidate) - not [cache-first](https://developer.chrome.com/docs/workbox/modules/workbox-strategies/#cache-first-cache-falling-back-to-network) as cache first will never update the cache.

If the cache is never updated then users will never get updates.
So unless you have an option to set an interval for checking/updating the cache we should use stale-while-revalidate.

Reply:
Page loads using stale-while-revalidate should be just as fast as page loads using cache-first, right?